src/formio/components/TimeField.js

```diff
@@ -1,15 +1,17 @@
-import {Formio} from "react-formio";
+import {Formio} from 'react-formio';
 import {getBEMClassName} from '../../utils';
+import MinMaxTimeValidator from 'formio/validators/MinMaxTimeValidator';
```

Comment:
```suggestion
import {Formio} from 'react-formio';

import MinMaxTimeValidator from 'formio/validators/MinMaxTimeValidator';
import {getBEMClassName} from 'utils';
```
let's tidy up our imports :)